osf_tests/test_external_accounts.py

```diff
@@ -39,3 +42,88 @@ def test_database_is_encrypted(self):
             row = cursor.fetchone()
             for blicky in row:
                 assert jwe.decrypt(bytes(blicky[len(EncryptedTextField.prefix):]), SENSITIVE_DATA_KEY) == self.magic_string
+
+
+class TestEncryptedTextField(TestCase):
```

Comment:
Minor: We aren't using `unittest.TestCase` for `osf_tests`, since it is unnecessary with pytest.

```python
class TestEncryptedTextField:
    @pytest.fixture
    def field(self):
        return EncryptedTextField(null=True, blank=True)

    def test_ensure_bytes...(self, field):
        # ...
```